src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Histograms produced by this instrumentation package specify explicit bucket boundaries.
```

Comment:
I think we could level this up a bit.

* The `http.server.request.duration` histogram (measured in seconds) produced by the metrics instrumentation in this package now uses the [Advice API](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/customizing-the-sdk/README.md#explicit-bucket-histogram-aggregation) to set default explicit buckets following the [OpenTelemetry Specification](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration).

Reply:
Fixed, with stable links ffee25f1a87cd74ccfc7aa105afe5f9c550c9787

Reply:
I approved but I don't know if I _love_ the stable link part 😄 Fine for the semantic conventions. But the link to advice API doc in main repo, I'm hoping to update that if this ever lands: https://github.com/dotnet/docs/pull/43640